products/extensions/b2b-components/shopping-lists/guides/entities-and-schema.md

```diff
@@ -0,0 +1,40 @@
+---
+nav:
+  title: Entities & Schema
+  position: 20
+
+---
+
+# Entities and schema
+
+## Entities
+
+### Shopping Lists
+
+Shopping lists represent a list of products prepared for a customer or a sales channel. They show basic information about the product list, such as the name, customer, sales channel and so on.
+
+### Line item
+
+The Shopping lists line item represents a set of products. These products are line items of the shopping list.
```

Comment:
This sentence seems to be quite unclear / overlapping with its meaning. Can it be rephrased ?

Reply:
Maybe: The Shopping List Line Item represents individual products within a shopping list. Each product in the shopping list is considered a line item.